test/acceptance/tests/connect/connect_inject_test.go

```diff
@@ -76,10 +95,8 @@ func TestConnectInject(t *testing.T) {
 					k8s.CheckStaticServerConnectionFailing(t, ctx.KubectlOptions(t), "http://localhost:1234")
 				}
 
-				consulClient := consulCluster.SetupConsulClient(t, true)
-
 				logger.Log(t, "creating intention")
-				_, _, err := consulClient.Connect().IntentionCreate(&api.Intention{
+				_, err := consulClient.Connect().IntentionUpsert(&api.Intention{
```

Comment:
Kinda curious about this one, for my own knowledge why are we doing upsert instead of create?

Reply:
`IntentionCreate` is [deprecated](https://github.com/hashicorp/consul/blob/21e855d860865f1472b3fdcb64a916e92bc83a9f/api/connect_intention.go#L376). Sorry I should have commented on the PR.